### DISCUSSION TITLE
Different output-files (names) per profile?

### DISCUSSION THREAD

**Discussion by cathblatter** [quarto-dev/quarto-cli#2802] [Q&A] [answered]
I use profiles to conditionally include parts (solutions to questions) in an PDF doc (teaching context, with Question as base profile and _quarto-solutions.yml). Content-wise everything works, but I tried setting `output-file: ex-solutions` to distinguish the plain file from the one with the solutions, but this parameter gets ignored (and the same file gets overwritten in the output-dir). Is there a way to define file-names per profile in the .yml or am I missing smth obvious? Thanks

**Comment by cathblatter** [comment, marked as answer]
edit: using `output-dir: solutions` to distinguish instead works like a charm

**Comment by juliandehne** [comment]
I still think that there should be a way to specify the filename in the profile

**Reply by mcanouil** [reply]
Use `output-file` under the `format.NAME` in your profile configuration file. It should work.

Note that this discussion is pretty old so very unlikely to be up to date with Quarto current abilities.
Prefer opening a new discussion rather than reviving very old ones.
